### DISCUSSION TITLE
[Question][Hw4] About seed_1 in json_seed

### DISCUSSION THREAD

**Discussion by ee12ha0220** [prosyslab-classroom/cs348-information-security#561] [Q&A]
My fuzzer crashes if I give seed_1.json as an input. But from what I understand about seed corpus, it has to contain only valid inputs. Am I getting something wrong? Thank you :)

**Comment by Standchen** [comment]
The input in seed_1.json results in `Divide-by-zero detected at line 492 and col 43` (with instrumented binary), and you can see that it is intended divide-by-zero statement.

**Comment by ee12ha0220** [comment]
Yes I can see that. But as I mutate seed_1.json and run the program, it keeps on generating crash and in output_dir/crash there is like 40 files generated when I run the code for 1 second. Am I doing the right thing?

**Comment by Standchen** [comment]
You'll get tens of thousands crashes as your fuzzer runs I guess, since mine does too.

**Comment by ee12ha0220** [comment]
Oh I see, then how should I specify a unique crash?

**Comment by Standchen** [comment]
In my case, I just crafted simple python/bash scripts to run each input in `output_dir/crash/*`.

**Comment by ee12ha0220** [comment]
Thanks a lot!
